scripts/jenkins/cloud/manual/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM registry.suse.de/suse/containers/sle-server/12-sp4/containers/suse/sles12sp4
```

Comment:
Just a thought, but have you considered building this container in IBS instead (e.g. in `Devel:Cloud:Images`) ?

Reply:
Yes, this thought occurred to me, but I have yet to understand exactly how one would do that. I'll address it though, because I think you make a good point that it would be better if the container was built there.